recipe/patches/0008-bpo-22699-Allow-compiling-on-debian-ubuntu-with-a-di.patch

```diff
@@ -8,19 +8,18 @@ This PR fixes one issue mentioned in the bpo
 https://bugs.python.org/issue22699#msg364685 with a slightly better
 patch than given
 ---
- setup.py | 24 +++++++++++++++++++++++-
- 1 file changed, 23 insertions(+), 1 deletion(-)
+ setup.py | 22 +++++++++++++++++++++-
+ 1 file changed, 21 insertions(+), 1 deletion(-)
 
 diff --git a/setup.py b/setup.py
-index 36b56ad034..178fa88029 100644
+index 36b56ad034..5a98b875d7 100644
 --- a/setup.py
 +++ b/setup.py
-@@ -636,9 +636,31 @@ class PyBuildExt(build_ext):
+@@ -636,9 +636,29 @@ class PyBuildExt(build_ext):
      def add_multiarch_paths(self):
          # Debian/Ubuntu multiarch support.
          # https://wiki.ubuntu.com/MultiarchSpec
 -        tmpfile = os.path.join(self.build_temp, 'multiarch')
-+        cc = sysconfig.get_config_var('CC')
```

Comment:
I got confused by this line in the previous PR. Removed it again from the patch as in 3.9 there is a global variable in `setup.py` named `CC`. I guess this is different in 3.10 and I mixed those up.